CAAppKitBridge/Source/NSView+CAMethods.h

```diff
@@ -25,26 +25,22 @@
 */
 
 #import <AppKit/AppKit.h>
+#import <Foundation/Foundation.h>
 #import <QuartzCore/QuartzCore.h>
-#import <GNUstepBase/GSVersionMacros.h>
 
-@interface  NSView (NSViewCAmethods)
+@interface  NSView (CAmethods)
 @property BOOL wantsLayer;
 - (CALayer *) makeBackingLayer;
 
 /* GNUstep specific methods */
-- (BOOL) addCARenderer: (CARenderer*) CARenderer;
+- (BOOL) addCARenderer: (CARenderer*)renderer;
```

Comment:
Consider `_addCARenderer:` or even `_gsAddCARenderer` to ensure this private method is less likely to conflict with user code.

Same comment for `removeCARenderer`.